packages/dds/tree/src/test/forestTestSuite.ts

```diff
@@ -1133,6 +1133,7 @@ export function testForest(config: ForestTestConfiguration): void {
 			});
 			it("when moving the last node in the field", () => {
 				const schemaFactory = new SchemaFactory("moving");
+				// TODO: this test does not seem to be in schema. Its moving from a required field to another required field.
```

Comment:
Its in a test, only impacts this specific code so anyone impacted by it can see the TODO making having external tracking for it very low value. Prabably more than half our 480+ tree TODOs are higher priority. I don't think adding it to our bug backlog would be helpful (its not a user impacting issue) and we are not tracking tech debt items this small individually.

Also this code is guaranteed to get revisited soon as part of removing typedJsonCursor (which I tried to do here, but failed since the schema and code don't align)